src/platform/ESP32/ConnectivityManagerImpl_WiFi.cpp

```diff
@@ -999,6 +999,7 @@ void ConnectivityManagerImpl::UpdateInternetConnectivityState(void)
                 // If the station interface has been assigned an IPv4 address, and has
                 // an IPv4 gateway, then presume that the device has IPv4 Internet
                 // connectivity.
+#if LWIP_IPV4
```

Comment:
we also have `CHIP_DEVICE_CONFIG_ENABLE_IPV4` through CHIPDevicePlatformConfig.h

Reply:
The option `CHIP_DEVICE_CONFIG_ENABLE_IPV4` is not used in case of ESP32 CHIPDevicePlatformConfig.h.